App/NavigationRoot.js

```diff
@@ -17,7 +17,7 @@ const NavigationRoot = createSwitchNavigator(
     Survey: SurveyStack,
   },
   {
-    initialRouteName: 'Auth',
+    initialRouteName: 'Survey',
```

Comment:
Shouldn't this still be auth as the initial route? We need an account to save the survey, don't we? and later, we will also implement navigation to let you log in and go to home screen if you have already taken the survey